src/pages/iou/request/step/withWritableReportOrNotFound.tsx

```diff
@@ -0,0 +1,49 @@
+import type {ComponentType, ForwardedRef} from 'react';
+import React, {forwardRef} from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import getComponentDisplayName from '@libs/getComponentDisplayName';
+import * as ReportUtils from '@libs/ReportUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Report} from '@src/types/onyx';
+
+type WithWritableReportOrNotFoundOnyxProps = {
+    /** The report corresponding to the reportID in the route params */
+    report: OnyxEntry<Report>;
+};
+
+type WithWritableReportOrNotFoundProps = WithWritableReportOrNotFoundOnyxProps & {
+    route: {params: {iouType: string; reportID: string} | undefined};
+};
+
+export default function <TRef, TProps extends WithWritableReportOrNotFoundProps>(WrappedComponent: ComponentType<TProps>) {
+    // eslint-disable-next-line rulesdir/no-negated-variables
```

Comment:
```suggestion
```

This rule looks unused

Reply:
@fabioh8010 If I disable that rule, it will throw an error because of the `Not` present in the name